tsconfig.json

```diff
@@ -16,5 +16,6 @@
     "noImplicitThis": false,
     "baseUrl": "./"
   },
-  "include": ["packages/*/src/**/*"]
+  "include": ["packages/**/*"],
```

Comment:
Can you explain why you've decided to do this?

Reply:
This will allow us to catch typing problems by using `yarn typecheck` without running tests.
No problem to revert it to the previous version 😃